classes/app.rb

```diff
@@ -0,0 +1,159 @@
+require_relative 'book'
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+
+class App
```

Comment:
You made a great effort to make this project! However, some errors need to be filled in!